advanced/dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -48,6 +48,7 @@ import {
   DEFAULT_MULTIVERSX_METHODS,
   DEFAULT_TRON_METHODS,
   DEFAULT_TEZOS_METHODS,
+  TEZOS_SAMPLE_KINDS,
```

Comment:
Could we bundle `TEZOS_SAMPLE_KINDS` and `DEFAULT_TEZOS_METHODS` into single prop `DEFAULT_TEZOS_METHODS` so we can keep the current app convention and reduce complexity of handling two method sources

Reply:
@ganchoradkov, Tezos has default methods (`tezos_getAccounts`, `tezos_sign`, [tezos_send](url)) which are used when establishing a connection.
the tezos_send method is used nearly for all possible Tezos operations: `transaction`, `origination`, `delegation`, stacking, calling smart contracts, etc. So to make an example for Tezos I added a number of sample operations to show how they should look like.
As a result, I have 2 lists of operations: 
 - `DEFAULT_TEZOS_METHODS` for 3 allowed methods
 - `TEZOS_SAMPLE_KINDS ` a list of methods extended with multiple kinds for `tezos_send`. This list is used to show the list of buttons for Tezos operations.

I bundled them now. Hope it looks better now